projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1240,6 +1254,18 @@ public function get_related_post_data_for_post( $post_id, $position, $origin ) {
 				$post->ID
 			),
 		);
+
+		/**
+		 * Filter the array of related posts.
+		 *
+		 * @module related-posts
+		 *
+		 * @since 2.8.0
+		 *
+		 * @param array $results Array of related posts.
+		 * @param string $post_id Post ID of the post for which we are retrieving Related Posts.
```

Comment:
`$post_id` is defined as an integer above:
https://github.com/Automattic/jetpack/blob/update/related-posts-display-filter/projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php#L1194

```suggestion
		 * @param int $post_id Post ID of the post for which we are retrieving Related Posts.
```